airbyte-integrations/connectors/source-pivotal-tracker/unit_tests/test_streams.py

```diff
@@ -2,101 +2,133 @@
-        json=project_memberships_response,
-    )
-    stream = ProjectMemberships(**project_args)
-    records = [r for r in stream.read_records(SyncMode.full_refresh, None, stream_slice, None)]
-    assert len(records) == 7
-    assert len(responses.calls) == 1
-
-
-@responses.activate
-def test_activity_stream(activity_response):
-    responses.add(
-        responses.GET,
-        "https://www.pivotaltracker.com/services/v5/projects/99/activity",
-        json=activity_response,
-    )
-    stream = Activity(**project_args)
-    records = [r for r in stream.read_records(SyncMode.full_refresh, None, stream_slice, None)]
-    assert len(records) == 15
-    assert len(responses.calls) == 1
-
-
-@responses.activate
-def test_labels_stream(labels_response):
-    responses.add(
-        responses.GET,
-        "https://www.pivotaltracker.com/services/v5/projects/99/labels",
-        json=labels_response,
-    )
-    stream = Labels(**project_args)
-    records = [r for r in stream.read_records(SyncMode.full_refresh, None, stream_slice, None)]
-    assert len(records) == 10
-    assert len(responses.calls) == 1
-
-
-@responses.activate
-def test_releases_stream(releases_response):
-    responses.add(
-        responses.GET,
-        "https://www.pivotaltracker.com/services/v5/projects/99/releases",
-        json=releases_response,
-    )
-    stream = Releases(**project_args)
-    records = [r for r in stream.read_records(SyncMode.full_refresh, None, stream_slice, None)]
-    assert len(records) == 1
-    assert len(responses.calls) == 1
-
-
-@responses.activate
-def test_epics_stream(epics_response):
-    responses.add(
-        responses.GET,
-        "https://www.pivotaltracker.com/services/v5/projects/99/epics",
-        json=epics_response,
-    )
-    stream = Epics(**project_args)
-    records = [r for r in stream.read_records(SyncMode.full_refresh, None, stream_slice, None)]
-    assert len(records) == 5
-    assert len(responses.calls) == 1
-
-
-@responses.activate
-def test_stories_stream(stories_response):
-    responses.add(
-        responses.GET,
-        "https://www.pivotaltracker.com/services/v5/projects/99/stories",
-        json=stories_response,
-    )
-    stream = Stories(**project_args)
-    records = [r for r in stream.read_records(SyncMode.full_refresh, None, stream_slice, None)]
-    assert len(records) == 4
-    assert len(responses.calls) == 1
+def load_file(fn):
+    return open(os.path.join("unit_tests", "responses", fn)).read()
+
+
+def get_stream_by_name(stream_name: str, config: Mapping[str, Any]) -> Stream:
+    source = SourcePivotalTracker()
+    matches_by_name = [stream_config for stream_config in source.streams(config) if stream_config.name == stream_name]
+    if not matches_by_name:
+        raise ValueError("Please provide a valid stream name.")
+    return matches_by_name[0]
+
+
+projects_mock = json.loads(load_file("projects.json"))
```

Comment:
Please use existing fixtures from the `conftest.py` for all tests in this module and delete these mocks